incubator/hnc/pkg/controllers/setup.go

```diff
@@ -61,3 +77,21 @@ func createObjectReconciler(newObjectController bool, mgr ctrl.Manager, f *fores
 	}
 	return or, or.SetupWithManager(mgr)
 }
+
+// LogActivity generates logs for performance testing.
+func LogActivity() {
```

Comment:
We opened issue #111 to publish metrics, though we were thinking to use OpenTelemetry instead of Prometheus, since OT's all about publishing metrics to various consumers, while Prometheus (I thought) was mainly a single implementation. If you have thoughts on OT vs Prom, maybe leave them in that issue?

But in the case of this one change, I think it will be faster to just read the logs while running performance tests - for starters, we don't have to install Prometheus, which is especially useful if we're testing on KinD or a small cluster. Does that sound reasonable? We can always update this later.

Reply:
Yeah, nothing to hold up this PR. We can move the convo to the ticket.